components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/MessageProcessorResource.java

```diff
@@ -224,12 +224,12 @@ private void changeProcessorStatus(MessageContext messageContext, JsonObject jso
                 messageProcessor.activate();
                 jsonResponse.put(Constants.MESSAGE_JSON_ATTRIBUTE, processorName + " : is activated");
             } else {
-                jsonResponse.put(Constants.MESSAGE_JSON_ATTRIBUTE, "Provided state is not valid");
+                jsonResponse = Utils.createJsonError("Provided state is not valid", axis2MessageContext, "400");
```

Comment:
Let's re-use the constants defined for these error codes

Reply:
+1. Done